src/battle_ai_main.c

```diff
@@ -2176,6 +2179,17 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
             else if (IsAbilityOfRating(aiData->abilities[battlerAtk], 5))
                 ADJUST_SCORE(-4);
             break;
+        case EFFECT_DOODLE: // Same as Role Play, but also check if the partner's ability should be replaced
+            if (aiData->abilities[battlerAtk] == aiData->abilities[battlerDef]
+              || aiData->abilities[BATTLE_PARTNER(battlerAtk)] == aiData->abilities[battlerDef]
+              || aiData->abilities[battlerDef] == ABILITY_NONE
+              || gAbilitiesInfo[aiData->abilities[battlerAtk]].cantBeSuppressed
+              || gAbilitiesInfo[aiData->abilities[BATTLE_PARTNER(battlerAtk)]].cantBeSuppressed
+              || gAbilitiesInfo[aiData->abilities[battlerDef]].cantBeCopied)
```

Comment:
```suggestion
            if (aiData->abilities[battlerAtk] == aiData->abilities[battlerDef]
             || aiData->abilities[BATTLE_PARTNER(battlerAtk)] == aiData->abilities[battlerDef]
             || aiData->abilities[battlerDef] == ABILITY_NONE
             || gAbilitiesInfo[aiData->abilities[battlerAtk]].cantBeSuppressed
             || gAbilitiesInfo[aiData->abilities[BATTLE_PARTNER(battlerAtk)]].cantBeSuppressed
             || gAbilitiesInfo[aiData->abilities[battlerDef]].cantBeCopied)
```

Reply:
Its not like this for any other line in this file

Reply:
Doesn't matter. If you make changes to it we might as well do proper alignment.

Reply:
:/ OK